### PR TITLE
Creation of migrations and modifications to various parts of the system

### LINKED ISSUE #289: [#287] Create and modify migration files

### LINKED ISSUE #290: [#287] Modify Models

### PR DESCRIPTION
**Pull request by shin5446**
## Content
As the title says
## Related Issue
fixed #289 
fixed #290

### CODE REVIEW

File: src/database/seeders/HostTableSeeder.php
Comment (akaka0039) [inline src/database/seeders/HostTableSeeder.php:1]: Could you change a file name like a 'RolesTableSeeder.php' ?

File: src/app/Models/SocialAccount.php [hunk 1]
```diff
@@ -15,13 +14,4 @@ class SocialAccount extends Model
...
     ];
-
-    /**
-     *
-     * @return BelongsToMany
-     */
-    public function users(): BelongsToMany
-    {
-        return $this->belongsToMany(User::class, 'user_id');
-    }
```
Comment (shin5446) [inline src/app/Models/SocialAccount.php:26]: I realize that this definition is not used anywhere, so I deleted it. Is that no problem? 
If you want to express one-to-many, can be like below
`return $this->belongsToMany(User::class, 'user_id');`
Maybe I reviewed it before, but I missed it.

Comment (akaka0039) [reply src/app/Models/SocialAccount.php:26]: no there is no problem
you can delete it

File: src/database/seeders/EventTeamsTableSeeder.php
Comment (akaka0039) [inline src/database/seeders/EventTeamsTableSeeder.php:1]: Great work!!!
Thank you for fix seeder files too

**Comment by akaka0039** [comment]
you don't have to care about [this ticket](https://github.com/users/akaka0039/projects/32/views/1?pane=issue&itemId=36687221)
In this future, we need to fix it

**Review by akaka0039 — APPROVED** [review]
I think this PR is enough to achieve the goal
Thank you so much!

**Comment by shin5446** [comment]
> you don't have to care about [this ticket](https://github.com/users/akaka0039/projects/32/views/1?pane=issue&itemId=36687221) In this future, we need to fix it

I'm aware of this. This will fix, when I develop backend logic for this! So don't worry.

**Pull request merged**